ncm-symlink/src/main/pan/components/symlink/schema.pan

```diff
@@ -2,15 +2,15 @@ ${componentschema}
 include 'quattor/types/component';
 
 type structure_symlink_replace_option_entry = {
-    @{use when renaming a given object type or to enable replacement for a specific object type.}
+    @{Use when renaming a given object type or to enable replacement for a specific object type.}
```

Comment:
I'm not sure this really explains what the option does — at least, I don't understand it.

Reply:
@jrha you are right to complain about this. To be honest: the original documentation in the pod file also makes little sense to me:
```
The existing object type. Valid values are: C<all>, C<dir>, C<dirempty>,
C<file>, C<link>, C<none>.  C<dirempty> means an empty directory only,
C<dir> means any directory. C<all> and C<none> are mutually exclusive
but can be combined with other object types to define the extension
to use when renaming a given object type or to prevent/enable replacement
for a specific object type.
```

Any suggestion on how to describe `all` and `none` correctly?

Reply:
Ugh... nope, might have to read the code and then rewrite the documentation.

Reply:
Opened an issue (#1363) to solve this for 19.2.